Loop/Managers/NotificationManager.swift

```diff
@@ -131,7 +131,7 @@ struct NotificationManager {
         // Give a little extra time for a loop-in-progress to complete
         let gracePeriod = TimeInterval(minutes: 0.5)
 
-        for minutes: Double in [20, 40, 60, 120] {
+        for (minutes, isCritical) in [(20.0, false), (40.0, false), (60.0, true), (120.0, true)] {
```

Comment:
DIY users typically do not have critical alerts entitlements enabled. Also, some users of Tidepool Loop may not grant access to critical alerts. Do the 60 and 120m notifications still post in those cases?

Reply:
@ps2 ah, good question.  I will test that

Reply:
@ps2 I just tested the case where the user does not grant access to critical alerts, and the alert (notification) still is presented, just not "critical".  I have not tried what happens in the case of not having the entitlement.  Do you think I should test that as well?
Also: I realize I should probably add a check for the feature flag, and for iOS > 12

Reply:
@ps2 Follow-up: I tested this code in the DIY app and confirmed: it also behaves as if you didn't permit critical alerts, that is, it still notifies, just not critically.